gipc/gipc.py

```diff
@@ -56,6 +57,18 @@
 log = logging.getLogger("gipc")
 
 
+# Python 3.8 changed the default subprocess method from 
+# fork to spawn. Spawn does not copy any state from the
+# old process. So we should force forking.
+try:
+    multiprocessing.set_start_method('fork')
+except:
```

Comment:
If this code would after all be merged: if we expect a specific exception, we should catch just that :) -- and the code comment should make pretty clear on which platforms this exception is expected to be raised and why!